common-design-patterns/global-header.html

```diff
@@ -1,369 +1,485 @@
+        <li role="none"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/defence.html">National security and defence</a></li>
+        <li role="none"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/culture.html">Culture, history and sport</a></li>
+        <li role="none"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/policing.html">Policing, justice and emergencies</a></li>
+        <li role="none"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/transport.html">Transport and infrastructure</a></li>
+        <li role="none"><a role="menuitem" tabindex="-1" href="http://international.gc.ca/world-monde/index.aspx?lang=eng">Canada and the world</a></li>
+        <li role="none"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/finance.html">Money and finances</a></li>
+        <li role="none"><a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/finance.html">Science and innovation</a></li>
+      </ul>
+    </div>
+  </nav>
+  <nav id="wb-bc" property="breadcrumb">
+    <h2>You are here:</h2>
+    <div class="container">
+      <ol class="breadcrumb">
+        <li><a href="https://www.canada.ca/en.html">Canada.ca</a></li>
+        <li><a href="https://www.canada.ca/en/government/about.html">About Canada.ca</a></li>
+        <li><a href="https://www.canada.ca/en/government/about/design-system.html">Canada.ca design system</a></li>
+        <li><a href="https://www.canada.ca/en/government/about/design-system/pattern-library.html">Template and pattern library for Canada.ca</a></li>
+      </ol>
+    </div>
+  </nav>
+</header>
+<main role="main" property="mainContentOfPage" class="container wb-prettify all-pre">
+<h1 property="name" id="wb-cont" dir="ltr"><span class="stacked"><span>Global header - Canada.ca design pattern</span>: <span>Canada.ca design system</span></span></h1>
+<div class="row">
+  <div class="col-md-12 pull-left">
+    <ul class="list-inline small mrgn-bttm-sm" style="line-height:1.65em" id="list-inline-desktop-only">
+      <li class="mrgn-rght-lg"> Last updated: 2022-08-17</li>
+    </ul>
+  </div>
+</div>
+<p><span class="label label-danger">Mandatory on standard pages</span></p>
+<p>The global header is at the top of each web page from the Government of Canada.</p>
+<p>Having the same global header on all pages:</p>
+<ul>
+  <li>strengthens the Canada.ca brand</li>
+  <li>promotes trust</li>
+  <li>provides a unified experience on the Government of Canada web presence</li>
+  <li>allows navigation across the broad range of services and information offered</li>
+</ul>
+<div class="pattern-demo mrgn-tp-lg">
+  <figure class="mrgn-bttm-lg">
+    <figcaption><b>Global header – large screen</b></figcaption>
+    <img src="../images/sign-in-desktop-en.jpg" class="img-responsive" alt="Diagram of global header for large screens. Text version below:">
```

Comment:
Is the sign-in-desktop-en.jpg file already in the repo? I've noticed that you push it in French but not in the English.